opencensus/stats/view.h

```diff
@@ -48,8 +50,8 @@ class View {
   // Returns true if this object is valid and data can be collected.
   bool IsValid() const;
 
-  // Returns a snapshot of the View's data.
-  const ViewData GetData();
+  // Returns a snapshot of the View's data with the given time.
+  const ViewData GetData(absl::Time now = absl::Now());
```

Comment:
Do we need the parameter of the time here? I think we should always use Now() because the value represents the value until now.

Reply:
I wanted it both for consistency and as an optimization: call `absl::Now()` once in the `StatsExporter` and use the same time for all views.

Reply:
Reverted this.